### PR TITLE
Feature/clean up geo areas

### PR DESCRIPTION
**Pull request by AdityaKhatri**
## Changes

* Clean geo areas page styling

## This PR doesn't introduce any:

- [x] temporary files, auto-generated files or secret keys
- [x] build works
- [x] eslint issues
- [x] typescript issues
- [x] `console.log` meant for debugging
- [x] typos
- [x] unwanted comments
- [x] conflict markers

## This PR contains valid:

- [x] permission checks
- [x] translations

### CODE REVIEW

File: app/views/ProjectEdit/GeoAreas/RegionsMap/styles.css [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+                    align-self: stretch;
+                }
+            }
+        }
+    }
+
+    .region {
+        position: absolute;
+        top: 90px;
+        left: 15px;
```
Comment (tnagorra) [inline app/views/ProjectEdit/GeoAreas/RegionsMap/styles.css]: We should define some variables or use existing ones

**Review by tnagorra — CHANGES_REQUESTED** [review]

**Review by tnagorra — DISMISSED** [review]
Please check deepscan issues

**Review by samshara — APPROVED** [review]

File: app/views/ProjectEdit/GeoAreas/RegionsMap/styles.css (hunk 1 above)
Comment (samshara) [reply app/views/ProjectEdit/GeoAreas/RegionsMap/styles.css]: @AdityaKhatri any thoughts?

Comment (AdityaKhatri) [reply app/views/ProjectEdit/GeoAreas/RegionsMap/styles.css]: I'll change this and send an update.

**Pull request merged**